src/js/test/rpc/server.test.ts

```diff
@@ -41,7 +41,6 @@ let sinonInstance: SinonSandbox;
 let server: ProcessBatchServer;
 let client: SupervisorClient;
 let writeError: SinonStub;
-let portNumber = 43000;
```

Comment:
Could you please rebase your branch.
`portNumber` is used inside tests. 
Also it is better use variable for magic constant like this, because if somebody change number in one place they can forgot change this number in another. on the other side if you have variable for that people need to change it in only one place.